src/app/games/lasca/tests/lasca.component.spec.ts

```diff
@@ -256,16 +256,46 @@ describe('LascaComponent', () => {
         }));
     });
     describe('experience as second player (reversed board)', () => {
+        it('should have first player on top in a reversed board', fakeAsync(async() => {
```

Comment:
voilà la redondance susmentionnée !
le "in a reversed board" est de trop :D #chipotage

Reply:
exactlyment